docs/reference/commandline/login.md

```diff
@@ -162,7 +164,7 @@ designated programs to handle credentials for *specific registries*. The default
 credential store (`credsStore` or the config file itself) will not be used for
 operations concerning credentials of the specified registries.
 
-### Logging out
+#### Usage
 
 If you are currently logged in, run `docker logout` to remove
 the credentials from the default store.
```

Comment:
If you're interested in making some additional changes (could be in a follow-up); I think we should include a link somewhere to the section describing where the cli configuration file is; https://github.com/docker/cli/blob/master/docs/reference/commandline/cli.md#configuration-files)

Reply:
I'll probably do that in a follow-up PR soon.